src/catalog/src/durable/upgrade/snapshots/objects_v48.txt

```diff
@@ -0,0 +1,100 @@
+CjwKOroBNwoJCgNrZXkSAggECh0KBGtpbmQSFUITU2VydmVyQ29uZmlndXJhdGlvbgoLCgV2YWx1ZRICCAQ=
```

Comment:
Could you remind me what these files are? Maybe serialized proto that is base64 encoded?

Reply:
Yep, they're serialized proto that is base64 encoded so that we can test that the serialization format doesn't change across versions.